docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -14,6 +14,93 @@ ms.service: multiple
 
 # Azure CLI 2.0 release notes
 
+## March 13, 2018
+
+Version 2.0.29
+
+### ACR
+
+* Added support for `--image` parameter to `repository delete`
+* Deprecated `--manifest` and `--tag` parameters of the `repository delete` command
+* Added `repository untag` command to remove a tag without deleting data
+
+### ACS
+
+* Added `aks upgrade-connector` command to upgrade an existing connector
+* Changed `kubectl` config files to use a more readable block-style YAML
+
+### Advisor
+
+* [BREAKING CHANGE] Renamed `advisor configuration get` to `advisor configuration list`
+* [BREAKING CHANGE] Renamed `advisor configuration set` to `advisor configuration update`
+* [BREAKING CHANGE] Removed `advisor recommendation generate` 
+* Added `--refresh` parameter to `advisor recommendation list`
+* Added `advisor recommendation show` command
+
+### Appservice
+
+* Deprecated `[webapp|functionapp] assign-identity`
+* webapp/functionapp: author managed identity commands `identity assign/show` [?]
+
+### Eventhubs
+
+* Initial release
+
+### Extension
+
+* Added check to warn user if used distro is different then the one stored in package source file, as this may lead into errors
+
+### Interactive
+
+* Persist history across different sessions
+* Fixed history while in scope [#]
+* Fixed progress meter for long running operations [#]
+* Completions more robust to command table exceptions [?]
+
+### Monitor
+
+* Deprecated the `monitor autoscale-settings` commands
+* Added `monitor autoscale` commands
+* Added `monitor autoscale profile` commands
+* Added `monitor autoscale rule` commands
+
+### Network
+
+* [BREAKING CHANGE] Removed `--tags` parameter from  `route-filter rule create`
+* Removed some erroneous default values for the following commands:
+  * `network express-route update`
+  * `network nsg rule update`
+  * `network public-ip update`
+  * `traffic manager profile update` [?]
+  * `network vnet-gateway update`
+* Added `network watcher connection-monitor` commands`
+* Added `--vnet` and `--subnet` parameters to `network watcher show-topology`
+
+### Profile
+
+* Deprecated `--msi` parameter for `az login`
+* Added `--identity` parameter for `az login` to replace `--msi`
+
+### RDBMS
+
+* [PREVIEW] Changed to use the API 2017-12-01-preview
+
+### Service Bus
+
+* Initial release
+
+### Storage
+
+* Fixed issue with missing endpoint suffix in `storage blob copy` [#] [?]
```

Comment:
Requires issue link, and not sure what command it refers to. Could not find a blob copy command.

Reply:
@williexu @troydai?

Reply:
https://github.com/Azure/azure-cli/issues/4971